notebooks/122-quantizing-model-with-accuracy-control/122-yolov8-quantization-with-accuracy-control.ipynb

```diff
@@ -0,0 +1,332 @@
+{
```

Comment:
<p>Could you add more information about ValidatorClass and purpose of data loader?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1234/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Added

Reply:
<p>sorry, I don't see, where ?</p>

Reply:
Something went wrong.  I've updated, please check again.